datahub-frontend/play.gradle

```diff
@@ -83,9 +82,6 @@ dependencies {
   annotationProcessor externalDependency.lombok
 }
 
-dist.dependsOn(':datahub-web-react:copyAssets')
```

Comment:
We were able to replace all of this? This is awesome!

Reply:
It just gets it in a different way now rather than a hacky copy task by depending on the project, I think.

Reply:
Slightly better, although technically this copy is only needed now for locally running via the run/ shell script and for unit tests. It is not needed for anything else, the distribution package gets the static asset from a -assets jar that is now generated from the react subproject.